plugins/module_utils/util.py

```diff
@@ -301,6 +301,28 @@ def fail(msg):
         labels[k] = to_text(v)
 
 
+def sanitize_labels_list(labels, labels_field, client=None, module=None):
+    def fail(msg):
+        if client is not None:
+            client.fail(msg)
+        if module is not None:
+            module.fail_json(msg=msg)
+        raise ValueError(msg)
+
+    if labels is None:
+        return
+    for index, label in enumerate(labels):
+        if not isinstance(label, string_types):
+            fail(
+                "The item {item!r} of {field} is not a string!".format(
+                    field=labels_field, item=label))
```

Comment:
In the other function this is checking the key, not the value. This isn't needed here.
```suggestion
```